typescript/infra/scripts/safes/create-safe.ts

```diff
@@ -54,10 +54,18 @@ async function main() {
   const safeAddress = await safe.getAddress();
 
   console.log(`Safe address: ${safeAddress}`);
-  console.log(
-    `Safe url: ${safeTxServiceUrl}/home?safe=${chain}:${safeAddress}`,
-  );
+  console.log(`Safe url: ${safeHomeUrl}/home?safe=${chain}:${safeAddress}`);
   console.log('url may not be correct, please check by following the link');
+
+  try {
+    const response = await fetch(`${safeHomeUrl}/home`);
+    const resultsJson = await response.json();
+    console.log(
+      `Add the transaction service url ${resultsJson.results.transactionService} as gnosisSafeTransactionServiceUrl to the metadata.yml in the registry`,
```

Comment:
I think `results` is an array and we'd need to filter for the specific chain for UIs that support multiple chains, like `https://safe-client.safe.global/v1/chains`

tbh if this feels like a big hassle to make it accurate, I think it's fine to just leave the programmatic transaction service finding to a later time